lib/internal/util.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const prefix = '(node) ';
+const prefix = `(node:${process.pid}) `;
```

Comment:
Perhaps consider using [`process.title`](https://nodejs.org/api/process.html#process_process_title) or [`process.release.name`](https://nodejs.org/api/process.html#process_process_release)?

Would make this a bit nicer for embedders. I'm not sure how `process.title` actually gets set though, so I'm not sure if it should be used for this?